src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -90,22 +91,23 @@ object DockerPlugin extends AutoPlugin {
       ),
       dockerBuildCommand := dockerExecCommand.value ++ Seq("build") ++ dockerBuildOptions.value ++ Seq("."),
       dockerCommands := {
-      val dockerBaseDirectory = (defaultLinuxInstallLocation in Docker).value
-      val user = (daemonUser in Docker).value
-      val group = (daemonGroup in Docker).value
-
-      val generalCommands = makeFrom(dockerBaseImage.value) +: makeMaintainer((maintainer in Docker).value).toSeq
-
-      generalCommands ++ Seq(
-        makeWorkdir(dockerBaseDirectory),
-        makeAdd(dockerBaseDirectory),
-        makeChown(user, group, "." :: Nil)
-      ) ++
-        makeExposePorts(dockerExposedPorts.value, dockerExposedUdpPorts.value) ++
-        makeVolumes(dockerExposedVolumes.value, user, group) ++
-        Seq(makeUser(user), makeEntrypoint(dockerEntrypoint.value), makeCmd(dockerCmd.value))
-
-    }
+        val dockerBaseDirectory = (defaultLinuxInstallLocation in Docker).value
+        val user = (daemonUser in Docker).value
+        val group = (daemonGroup in Docker).value
+
+        val generalCommands = makeFrom(dockerBaseImage.value) +: makeMaintainer((maintainer in Docker).value).toSeq
+
+        generalCommands ++
+          Seq(
+            makeWorkdir(dockerBaseDirectory),
+            makeAdd(dockerBaseDirectory),
+            makeChown(user, group, "." :: Nil)
+          ) ++
+          dockerLabels.value.map(makeLabel) ++
+          makeExposePorts(dockerExposedPorts.value, dockerExposedUdpPorts.value) ++
+          makeVolumes(dockerExposedVolumes.value, user, group) ++
+          Seq(makeUser(user), makeEntrypoint(dockerEntrypoint.value), makeCmd(dockerCmd.value))
+      }
```

Comment:
These look like formatting changes. When you run `sbt scalafmt` it should format correctly.
Or if you use intellij along with the `scalafmt` plugin you may use a newer or older version.